sdk/python/foundation-models/system/inference/automatic-speech-recognition/asr-batch-endpoint.ipynb

```diff
@@ -0,0 +1,484 @@
+    "#### Connect to workspace and registry using ML clients."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    credential = InteractiveBrowserCredential()\n",
+    "\n",
+    "workspace_ml_client = MLClient(\n",
+    "    credential,\n",
+    "    subscription_id=subscription_id,\n",
+    "    resource_group_name=resource_group_name,\n",
+    "    workspace_name=workspace_name,\n",
+    ")\n",
+    "# The models, fine tuning pipelines, and environments are available in the AzureML system registry, \"azureml\"\n",
+    "registry_ml_client = MLClient(credential, registry_name=\"azureml\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create a compute cluster.\n",
+    "Use the model card from the AzureML system registry to check the minimum required inferencing SKU, referenced as `size` below. If you already have a sufficient compute cluster, you can simply define the name in `compute_name` in the following code block."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "compute_name = \"cpu-cluster\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "compute_cluster = AmlCompute(\n",
+    "    name=compute_name,\n",
+    "    description=\"An AML compute cluster\",\n",
+    "    size=\"Standard_DS4_V2\",\n",
+    "    min_instances=0,\n",
+    "    max_instances=3,\n",
+    "    idle_time_before_scale_down=120,\n",
+    ")  # 120 seconds\n",
+    "\n",
+    "workspace_ml_client.begin_create_or_update(compute_cluster)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Pick a model to deploy\n",
+    "\n",
+    "Browse models in the Model Catalog in the AzureML Studio, filtering by the `automatic-speech-recognition` task. In this example, we use the `openai-whisper-large` model. If you have opened this notebook for a different model, replace the model name and version accordingly. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = \"openai-whisper-large\"\n",
+    "model_version = \"1\"\n",
+    "foundation_model = registry_ml_client.models.get(model_name, model_version)\n",
+    "print(\n",
+    "    f\"Using model name: {foundation_model.name}, version: {foundation_model.version}, id: {foundation_model.id} for inferencing.\"\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Prepare data for inference.\n",
```

Comment:
nit: same here. Some step titles are with trailing periods and some aren't.

Reply:
Good call. That can be done for all the notebooks in the next update.